packages/compiler-sfc/__tests__/__snapshots__/compileScriptPropsTransform.spec.ts.snap

```diff
@@ -96,6 +96,23 @@ return () => {}
 }"
 `;
 
+exports[`sfc props transform > default values w/ type declaration & key is string 1`] = `
+"import { defineComponent as _defineComponent } from 'vue'
+
+export default /*#__PURE__*/_defineComponent({
+  props: {
+    \\"foo\\": { type: Number, required: true, default: 1 }
```

Comment:
I think we can have an algorithm to remove redundant quotes even if users are using them.

Reply:
You are right, it's too late today, tomorrow I can see how to achieve it better😂

Reply:
I changed the logic so that the compiled result has no quotes